### PR TITLE
Address minor issues identified in 10/24 Code Audit

### PR DESCRIPTION
**Pull request by andrewlee94**
## Summary/Motivation:
The developer team did a code audit in October and identified a number of issues that should be investigated and resolved if possible. This PR addresses those issues that are easily fixed.

## Changes proposed in this PR:
- Update CODEOWNERS file in light of staffing changes
- Add condition to `xfail` marker on plate heat exchanger tests to only `xfail` on Windows system.

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

**Review by agarciadiego — APPROVED** [review]
Looks good to me

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -23,13 +23,13 @@ conftest.py @lbianchi-lbl
...
-/idaes/core/ @andrewlee94 @dallan-keylogic @lbianchi-lbl
+/idaes/core/ @agarciadiego @dallan-keylogic @lbianchi-lbl
 /idaes/core/dmf/ @dangunter
-/idaes/core/surrogate/ @andrewlee94 @bpaul4 @avdudchenko @rundxdi
+/idaes/core/surrogate/ @bpaul4 @avdudchenko @rundxdi
 /idaes/core/ui/ @dangunter
 
 # Models
-/idaes/models/ @andrewlee94 @bpaul4
+/idaes/models/ @agarciadiego @bpaul4
```
Comment (dallan-keylogic) [inline .github/CODEOWNERS:32]: Add me to `/idaes/models/property_models/`

Comment (andrewlee94) [inline .github/CODEOWNERS:32]:
```suggestion
/idaes/models/ @agarciadiego @bpaul4
/idaes/models/properties/ @agarciadiego @dallan-keylogic 
```

Comment (andrewlee94) [reply .github/CODEOWNERS:32]: Done.

File: idaes/models_extra/column_models/tests/test_plate_heat_exchanger.py [hunk 2]
```diff
@@ -14,15 +14,18 @@
 Tests for Plate Heat Exchnager  unit model.
 Author: Akula Paul
 """
-
+import sys
```
Comment (agarciadiego) [inline idaes/models_extra/column_models/tests/test_plate_heat_exchanger.py:17]: If the change was reverted is this import still necessary?

Comment (andrewlee94) [reply idaes/models_extra/column_models/tests/test_plate_heat_exchanger.py:17]: No, it is not.
```suggestion
```

**Review by dallan-keylogic — APPROVED** [review]
LGTM

**Pull request merged**